src/components/AttachmentCarousel/index.js

```diff
@@ -244,8 +244,8 @@ class AttachmentCarousel extends React.Component {
     renderCell(props) {
         // Use window width instead of layout width to address the issue in https://github.com/Expensify/App/issues/17760
         // considering horizontal margin and border width in centered modal
-        const modalStyles = styles.centeredModalStyles(this.props.isSmallScreenWidth);
-        const style = [props.style, styles.h100, {width: this.props.windowWidth - (modalStyles.marginHorizontal + modalStyles.borderWidth) * 2 + 1}];
+        const modalStyles = styles.centeredModalStyles(this.props.isSmallScreenWidth, true);
+        const style = [props.style, styles.h100, {width: PixelRatio.roundToNearestPixel(this.props.windowWidth - (modalStyles.marginHorizontal + modalStyles.borderWidth) * 2)}];
```

Comment:
>  In order to solve the problem of the decimal calculation accuracy of contentOffset on android chrome, the scroll position does not match the snapToInterval, so we need to use PixelRatio.roundToNearestPixel


If we don't use `PixelRatio.roundToNearestPixel`, it will cause the following situation on android chrome, you can print `event.nativeEvent.contentOffset` and `this.state.containerWidth` in `AttachmentCarousel.FlatList.onScroll` to compare the difference

<img src="https://github.com/Expensify/App/assets/20135674/e6ee012f-2ec5-40be-aee1-95c63607bab3" height=300 />